docs/cli.md

```diff
@@ -27,25 +27,24 @@ This document demonstrates how to use the CLI but also shows what happens in `KU
             * [Get the History to PlanExecutions](#get-the-history-to-planexecutions)
 
 
-## Kubectl KUDO Plugin
+## Setup the KUDO Kubectl Plugin
 
 ### Requirements
 
-- Kubectl version of `1.12.0` or later
-- GitHub set up
-  - GitHub OTP token in `$HOME/.git-credentials` OR
+- `kubectl` version `1.12.0` or newer
+- Configure GitHub authentication to be able to pull from [kudobuilder/frameworks](https://github.com/kudobuilder/frameworks).
```

Comment:
Is there a link with instructions in the GH documentation?

Reply:
since I just went thru this...   GH docs: https://git-scm.com/docs/git-credential-store
and likely:  https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line

Reply:
Thanks! Added those links